README.md

```diff
@@ -1,15 +1,18 @@
+<a href="https://imgbb.com/"><img src="https://i.ibb.co/gPybnRP/pngwing-com.png" alt="PPSSPP Original Logo" width="128" height="128"></a>
+<a href="https://imgbb.com/"><img src="https://i.ibb.co/8mFD49P/pngwing-com-1.png" alt="PPSSPP Gold Logo - Buy Now !" border="0" width="128" height="128"></a>
```

Comment:
```suggestion
```
If this pull request is merged, I would recommend one logo. In this case, the regular logo.